Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CA/Constants.h

```diff
@@ -0,0 +1,116 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file Constants.h
+/// \brief
+///
+
+#ifndef TRACKINGITSU_INCLUDE_CONSTANTS_H_
+#define TRACKINGITSU_INCLUDE_CONSTANTS_H_
+
+#include <climits>
+
+#include "ITSReconstruction/CA/Definitions.h"
+
+namespace o2
+{
+namespace ITS
+{
+namespace CA
+{
+
+namespace Constants
+{
+
+constexpr bool DoTimeBenchmarks = true;
+
+namespace Math
+{
+constexpr float Pi{ 3.14159265359f };
+constexpr float TwoPi{ 2.0f * Pi };
+constexpr float FloatMinThreshold{ 1e-20f };
+}
+
+namespace ITS
+{
+constexpr int LayersNumber{ 7 };
+constexpr int LayersNumberVertexer{ 3 };
+constexpr int TrackletsPerRoad{ 6 };
+constexpr int CellsPerRoad{ LayersNumber - 2 };
+constexpr int UnusedIndex{ -1 };
+constexpr float Resolution{ 0.0005f };
+
+GPU_HOST_DEVICE constexpr GPUArray<float, LayersNumber> LayersZCoordinate()
+{
+  return GPUArray<float, LayersNumber>{ { 16.333f, 16.333f, 16.333f, 42.140f, 42.140f, 73.745f, 73.745f } };
+}
+GPU_HOST_DEVICE constexpr GPUArray<float, LayersNumber> LayersRCoordinate()
+{
+  return GPUArray<float, LayersNumber>{ { 2.33959f, 3.14076f, 3.91924f, 19.6213f, 24.5597f, 34.388f, 39.3329f } };
+}
+}
+
+namespace Thresholds
+{
+GPU_DEVICE constexpr GPUArray<float, ITS::TrackletsPerRoad> TrackletMaxDeltaZThreshold()
+{
+  return GPUArray<float, ITS::TrackletsPerRoad>{ { 0.1f, 0.1f, 0.3f, 0.3f, 0.3f, 0.3f } };
+}
+constexpr float CellMaxDeltaTanLambdaThreshold{ 0.025f };
```

Comment:
same here and below: I would keep all tolerances settable

Reply:
You are right! Anyway I will have to do it when I introduce the second iteration and to perform a bit of cut tuning.